tools/wptrunner/wptrunner/wptcommandline.py

```diff
@@ -535,7 +535,7 @@ def check_args(kwargs):
 
     if kwargs['extra_prefs']:
         # If a single pref is passed in as a string, make it a list
-        if type(kwargs['extra_prefs']) in (str, unicode):
+        if isinstance(kwargs['extra_prefs'], (binary_type, text_type)):
```

Comment:
I know you changed this for @gsnedders comment, but did you check the actual types we get in py2/py3?

Reply:
With current code base, type(kwargs['extra_prefs']) returns <class 'list'> for both py2 & py3.

Reply:
I think we should check text type here. Just updated the code. Please let me have your thoughts. Thank you!